mgit/lib/resolver.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+
+/**
+ * Resolves repository for given package.
+ *
+ * @param {String} name Package name.
+ * @returns {String}
```

Comment:
May return `undefined` if a name isn't found. BTW, it'd be good to call this the `default-repository-resolver.js`. `lib/resolver.js` is too vague.